test/e2e/far_e2e_test.go

```diff
@@ -0,0 +1,64 @@
+package e2e
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/medik8s/fence-agents-remediation/api/v1alpha1"
+)
+
+const fenceAgentDummy = "echo"
+
+var _ = Describe("FAR E2e", func() {
+	Context("fence agent - dummy", func() {
+		var far *v1alpha1.FenceAgentsRemediation
+		testNodeName := "dummy-node"
+
+		BeforeEach(func() {
+			testShareParam := map[v1alpha1.ParameterName]string{}
+			testNodeParam := map[v1alpha1.ParameterName]map[v1alpha1.NodeName]string{}
+			far = createFAR(testNodeName, fenceAgentDummy, testShareParam, testNodeParam)
+		})
+
+		AfterEach(func() {
+			deleteFAR(far)
+		})
+
+		It("should check whether the CR has been created", func() {
+			testFarCR := &v1alpha1.FenceAgentsRemediation{}
+			Expect(k8sClient.Get(context.Background(), client.ObjectKeyFromObject(far), testFarCR)).To(Succeed(), "failed to get FAR CR")
+		})
+	})
+})
+
+// createFAR assigns the input to FenceAgentsRemediation object, creates CR it with offset, and returns the CR object
```

Comment:
is there a maybe a typo here? "creates CR it with offset"

Reply:
Nit: IIUC the offset refers to the error reporting and not the CR creation

Reply:
Correct I will fix it in a follow up